package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "yajsapi",
-  "version": "0.3.0",
+  "version": "0.3.1-alpha.4",
```

Please remember to change it to `0.3.1` before merging.
```suggestion
  "version": "0.3.1",
```